docs/networks/how_to_backup_and_recover_your_node.md

```diff
@@ -1,46 +1,64 @@
-# How To Backup And Recover Your Node
+# Backup And Recover Your validator
 
-It is pretty straight forward to setup a node when your old node when your old setup doesn't work anymore. You can simply
-run
+We made it simple and straight forward to recover a node with its validators when your old node doesn't work anymore or you want to transfer everything to a new machine.
 
-```bash
-lukso network init --chain l16
-```
+## The keystore
 
-in another instance and the setup is done in no time.
-
-The problem starts with the keystore. How do you copy and move the keystore to another instance.
-
-The keystore is determinstic derivative of 4 values
+The keystore consists the following 4 values:
 
 * **ValidatorMnemonic** - the seed for your validator keys
 * **WithdrawalMnemonic** - which is needed to withdraw your stakes
 * **KeystoreIndexFrom** and **KeyStoreIndexTo** - the key positions that you chose to create a keystore with x amount of validator keys.
 
-If you now these 4 values you can recover your validator setup.
+If you backup those 4 values and store them safely you can always recover your validator setup.
+
+## Keystore backup
 
-The LUKSO CLI offers 2 commands to deal with the recovery.
+First we explain how to backup and transfer the keystore files to another machine.
+
+The LUKSO CLI offers 2 commands to make a backup and recover this backup on a new machine.
+
+First we explain the backup command:
 
 ```bash
 lukso network validator backup
 ```
 
-This will produce a recovery file **node_recovery.json** with exactly these four values. Copy the content once you have setup a node and store it somewhere
-safe.
-
-If you want to recover the node you can simply do
+This will produce a file named: **"node_recovery.json"** with the 4 values of the keystore. Make a backup of your validator keystore with this command after you have succesfully setup your node and validators on your machine. Store this file somewhere safe and offline.
 
+## Start up your new node
 
+You always start on your new machine with installing the CLI and initializing the network before you can recover your old validators:
```

Comment:
Make sure you have the cli installed.....but I think it should not show this installation command as it might change and we will have a page for the installation of the CLI